README.md

```diff
@@ -187,6 +188,17 @@
 * [Simple Nixos Mailserver](https://gitlab.com/simple-nixos-mailserver/nixos-mailserver) - A complete mailserver, managed with NixOS modules.
 * [Stylix](https://github.com/danth/stylix) - System-wide colorscheming and typography for NixOS.
 
+## NixOS Configuration Editors
+
+### Desktop apps
+
+* [nixos-conf-editor](https://github.com/vlinkz/nixos-conf-editor) - Desktop app in Rust and GTK.
```

Comment:
Maybe use more descriptive blurb, something like “Modify values of options in NixOS configuration. Graphical editor using GTK and Rust.”